### ISSUE TITLE
Axios deprecated warnings

### ISSUE THREAD

**Issue by ojongerius** [epsagon/epsagon-node#511]
We started using the node library, and see our logs are getting polluted with warnings about transitional  options being used from the Axios library (during cold starts):

```
WARN [Axios v1.0.2] Transitional option 'clarifyTimeoutError' has been deprecated since v1.0.0 and will be removed in the near future
WARN [Axios v1.0.2] Transitional option 'forcedJSONParsing' has been deprecated since v1.0.0 and will be removed in the near future
WARN	[Axios v1.0.2] Transitional option 'silentJSONParsing' has been deprecated since v1.0.0 and will be removed in the near future | 2021-08-11T05:07:34.935Z f7f87242-ba72-45a1-a645-abe6beac26b0
```

This started in https://github.com/epsagon/epsagon-node/releases/tag/v1.113.2 which upgrades `axios` to `axios-minified` at version `1.0.2`. I verified the messages go away when we downgrade.

**Comment by amouly** [comment]
I have the same warning.

**Comment by sagivr2020** [comment]
Hi, fix can be found in epsagon node version: 1.114.1 (latest).
Please upgrade.
